### PR TITLE
feat(range) !: adding customization properties for range component

### PR DESCRIPTION
**Pull request by TheSnakeRekt**

### CODE REVIEW

**Review by pedrorebelocosta — APPROVED** [review]

File: packages/ui-components/src/components/range/range.base.scss [hunk 1]
```diff
@@ -1,18 +1,23 @@
 @import '../../assets/styles/globals';
 
+
 :host {
 	/**
 	 * @prop --range-width: Width of the range slider
+	 * @prop --range-height: Height of the range slider
+	 * @prop --button-selector-width: Width of the range thumb
+	 * @prop --button-selector-height: Height of the range thumb
 	 * @prop --slider-background-filled: color of the slider when its full
```
Comment (pedrorebelocosta) [inline packages/ui-components/src/components/range/range.base.scss:10]: rename this prop to `range` to keep consistency with the others ;)

File: packages/ui-components/src/components/range/range.base.scss [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
 	 * @prop --slider-background-empty: color of the slider when its empty
```
Comment (pedrorebelocosta) [inline packages/ui-components/src/components/range/range.base.scss:11]: same for this one ;)

File: packages/ui-components/src/components/range/range.base.scss (hunk 1 above)
Comment (pedrorebelocosta) [inline packages/ui-components/src/components/range/range.base.scss:10]: rename this prop's prefix to `range-` to keep consistency with the others ;)

**Review by pedrorebelocosta — CHANGES_REQUESTED** [review]

File: packages/ui-components/src/components/range/range.types.ts [hunk 3]
```diff
@@ -14,4 +14,6 @@ export interface IRange {
 	value?: number;
 	/** (optional) Range value step */
 	step?: number;
+	/** (optional) Show value label */
```
Comment (djcouto) [inline packages/ui-components/src/components/range/range.types.ts:17]: Hide value label? 😄 
Otherwise, change the prop name to `showLabel`

**Review by djcouto — CHANGES_REQUESTED** [review]
Besides what @pedrorebelocosta pointed out, just one more comment.

After that for me, you're good to go 👍

**Review by djcouto — APPROVED** [review]

File: packages/ui-components/src/components/range/range.tsx [hunk 4]
```diff
@@ -36,22 +41,22 @@ export class KvRange implements IRange, IRangeEvents {
 	private applyCssStyles = () => {
 		const rangeInputValue = this.getRangeElement();
 
-		const selector = this.el.shadowRoot.getElementById('select-value');
-
 		const inputValue = this.getInputValue(rangeInputValue);
 		const percentage = getInputPercentageFromValue(inputValue, this.min, this.max);
 		const inputOffSet = getInputOffset(percentage);
 		const valueOffset = getValueOffset(percentage, inputValue);
```
Comment (GoncaloCamaz) [inline packages/ui-components/src/components/range/range.tsx:47]: I believe this valueOffset is only used inside the !this.hideLabel if. If so, maybe would be better to move this calculation inside the if

File: packages/ui-components/src/components/range/range.base.scss [hunk 5]
```diff
@@ -3,16 +3,20 @@
 :host {
 	/**
 	 * @prop --range-width: Width of the range slider
+	 * @prop --range-height: Height of the range slider
+	 * @prop --range-selector-width: Width of the range thumb
```
Comment (ricardogoncalves89) [inline packages/ui-components/src/components/range/range.base.scss:7]: `--range-selector-width` and `--range-selector-height` can be merged into one like `--range-selector-radius`

**Review by GoncaloCamaz — APPROVED** [review]

**Review by ricardogoncalves89 — CHANGES_REQUESTED** [review]
See my comment and the commit should have the break reference because you rename some css variables and it can cause some issues in applications that already use theses variables

**Review by ricardogoncalves89 — APPROVED** [review]

**Review by pedrorebelocosta — APPROVED** [review]

**Comment by pedrorebelocosta** [comment]
Before merging, make sure your commit message is correct as there is a space between the commit type and description:

current:
`feat(range) !: adding customization properties for range component`

when it should be: 
`feat(range)!: adding customization properties for range component`

**Comment by miguelpinto98** [comment]
@TheSnakeRekt Please fix the commit message and rebase.

**Pull request merged**